CoroutineTemplate/app/src/main/java/co/nimblehq/coroutine/ui/ErrorMapping.kt

```diff
@@ -0,0 +1,9 @@
+package co.nimblehq.coroutine.ui
+
+import android.content.Context
+import co.nimblehq.coroutine.R
+
+fun Throwable.userReadableMessage(context: Context): String {
```

Comment:
@Tuubz hmm, I named it as `ErrorMapping` to map Error (in the domain module with use case) with error string resources (in the app module). In case we don't keep use case and Error in non-app module, we could name it as `ThrowableExt.kt`

Reply:
@luongvo @Tuubz I think we have plan for implementing domain module like we do in `RxJava` template in the future. So in this case we might want to keep this file as is 🤔

Reply:
@chornerman okay 👌